index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./assets/css/app.css">
+    <title>Emir Sağıt Personal Website</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="nav-flex">
+                <a href="#" class="nav-text">My Logo</a>
+                <button class="nav-button" type="button">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" stroke="#6070FF" viewBox="0 0 24 24">
+                        <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+                    </svg>
+                </button>
+            </div>
+        </nav>
+        <div class="header bg-header">
+            <h1 class="header-title">
+                I’m Mariane
+                Glad to see you!
+            </h1>
+            <p class="header-text">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <div class="social">
+                <p class="social-title">LET’S CONNECT</p>
```

Comment:
- [ ] The button is a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design), please fix it

Reply:
Well done!